.github/workflows/release.yml

```diff
@@ -8,6 +8,8 @@ on:
     paths:
       - 'releases/**'
 
+permissions: {}
```

Comment:
Release job actually releases and updates repo, i reckon it'll need at least write permissions for the content and release, and most likely pull requests/issues

Reply:
I sent an updated to add ~these. Is there a good way to test which are actually necessary?

Reply:
Not right now, since we dry run the release so its not actually doing anything on the repos themselves.

We could, in theory, test it with some "testing fork" but there will need to be quite extensive support, but then it wont really test the same behavior as the fork could diverge without us noticing :/